### ISSUE TITLE
Cannot import tobler due to import of deprecated numpy function

### ISSUE THREAD

**Issue by florianjehn** [pysal/tobler#166]
When I try to import tobler it fails with:
`ImportError: cannot import name 'asscalar' from 'numpy'`

This seems to be happening as asscalar is deprecated (https://numpy.org/doc/1.21/reference/generated/numpy.asscalar.html) and tobler not specifying which numpy version it needs.

**Comment by knaaptime** [comment]
thanks for the report. can you provide a full traceback? we don't use the `asscalar` function anywhere in tobler, so it may be an issue with one of the dependencies

**Comment by florianjehn** [comment]
Sure, here you go:

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 import tobler

File ~/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/tobler/__init__.py:7, in <module>
      1 __version__ = "0.5.4"
      2 """
      3 :mod:`tobler` --- A library for spatial interpolation
      4 =================================================
      5 
      6 """
----> 7 from . import area_weighted
      8 from . import dasymetric
      9 from . import model

File ~/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/tobler/area_weighted/__init__.py:1, in <module>
----> 1 from .area_interpolate import _area_interpolate_binning as area_interpolate
      2 from .area_interpolate import _area_interpolate as _slow_area_interpolate
      3 from .area_interpolate import _area_tables, _area_tables_binning, _area_tables_raster

File ~/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/tobler/area_weighted/area_interpolate.py:8, in <module>
      6 import numpy as np
      7 import geopandas as gpd
----> 8 from ._vectorized_raster_interpolation import _fast_append_profile_in_gdf
      9 import warnings
     10 from scipy.sparse import dok_matrix, diags

```
File ~/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/tobler/area_weighted/_vectorized_raster_interpolation.py:19, in <module>
     17 import rasterio
     18 from rasterio.mask import mask
---> 19 import rasterstats as rs
     20 import warnings
     21 from tqdm.auto import tqdm

File ~/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/rasterstats/__init__.py:3, in <module>
      1 # -*- coding: utf-8 -*-
      2 from .main import gen_zonal_stats, raster_stats, zonal_stats
----> 3 from .point import gen_point_query, point_query
      4 from rasterstats import cli
      5 from rasterstats._version import __version__

File ~/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/rasterstats/point.py:6, in <module>
      4 from shapely import wkt
      5 from numpy.ma import masked
----> 6 from numpy import asscalar
      7 from .io import read_features, Raster
     10 def point_window_unitxy(x, y, affine):

ImportError: cannot import name 'asscalar' from 'numpy' (/home/florian/local/anaconda3/envs/seaweed-growth-model/lib/python3.8/site-packages/numpy/__init__.py)

```

**Comment by martinfleis** [comment]
This is coming from `rasterstats` and it seems that it has been fixed on their side a long time ago. Which version of `rasterstats` do you have? Can you try updating to the latest one to see if that fixes the issue?

**Comment by florianjehn** [comment]
Fixed the issue. Thanks. I am using conda/mamba and it seems the newest version available there is 0.14, which does not have the fix yet.
